spec/system/post_index_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Index', type: :system do
+  before do
```

Comment:
- [ ] Please you are missing a test that tests you can see a section for pagination if there are more posts than fit on the view  :pray: